builder/deploy/index.ts

```diff
@@ -25,18 +25,27 @@ export default createBuilder<any>(
     }
 
     let buildResult: BuilderOutput;
-    const configuration = builderConfig.mode
-      ? builderConfig.mode
-      : "production";
+    if (builderConfig.noBuild) {
+      context.logger.info(`📦 Skipping build`);
+      buildResult = {
+        success : true
+      }
+    } else {
+      const configuration = builderConfig.configuration ? builderConfig.configuration : "production";
 
-    const build = await context.scheduleTarget({
-      target: "build",
-      project: context.target !== undefined ? context.target.project : "",
-      configuration
-    });
+      const overrides = {
+        // this is an example how to override the workspace set of options
+        ...(builderConfig.baseHref && { baseHref: builderConfig.baseHref })
+      };
```

Comment:
I think this is comming from the ngx-deploy-starter project, correct? I could not find any reason why this should be here? Is it because you would like to override default build properties?

Reply:
This is to build with baseHref, where we want to deploy apps under a context.